mappings/net/minecraft/class_12193.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_12193
```

Comment:
Please name this class

Reply:
I do not like this class very much.

Reply:
Should this not just be `net/minecraft/world/attribute/AttributeTypes`?